android/src/main/java/com/opentokreactnative/OTSubscriberViewNativeManager.kt

```diff
@@ -33,11 +34,20 @@ class OTSubscriberViewNativeManager(context: ReactApplicationContext) :
         view.setSessionId(sessionId)
     }
 
-
-    @ReactProp(name = "subscribeToAudio")
-    override public fun setSubscribeToAudio(view: OTSubscriberViewNative, value: Boolean) {
-        view.setSubscribeToAudio(value)
+    override fun setSubscribeToAudio(
+        view: OTSubscriberViewNative?,
+        value: Boolean
+    ) {
+        // TODO("Not yet implemented")
+        view?.setSubscribeToAudio(value)
     }
+    /*
```

Comment:
Questions about this commented out block:

* Why is `@ReactProp(name = "subscribeToAudio")` commented out, but `@ReactProp` calls for other props (like `@ReactProp(name = "audioVolume")`) are not? Should we move the `@ReactProp` calls out of this commented block and into other parts of the file?

* Can we remove the methods in this commented block that are implemented elsewhere in the code (like `setSubscribeToAudio()`, `setSubscribeToVideo()`, and `setSubscribeToCaptions()`)?

Reply:
@jeffswartz I don't think @ReactProp(name = "subscribeToAudio") is not required. Did you add it or auto generated. When I auto generated the code stub using Android Studio, the @ReactProp didn't get genrated.